README.md

```diff
@@ -1,2 +1,14 @@
-# chess
-Chess game with pygame library.
+# Chess
+Chess game with pygame library made from zero.
+Chess with an ai engine option.
+
+## On The Project:
+Thats was my first python project.
+I did this project because I love chess and i wanted to practice and test my python skills.
+
+## How To Use:
+You just need to downold pygame libary and python-3 and off course all the images and the program.
+
+## Images
```

Comment:
Add another section before images, with "How to bot/game engine works?", and explain shortly on the recursion, and position evaluation table, also explain shortly and add a link to mini max wikipedia page.

Reply:
:+1:

Reply:
:+1: